conans/test/functional/generators/pkg_config_test.py

```diff
@@ -203,3 +203,25 @@ def test_empty_include(self):
         pc = client.load("pkg.pc")
         self.assertNotIn("libdir=${prefix}/lib", pc)
         self.assertNotIn("includedir=${prefix}/include", pc)
+
+    def test_tuple_description(self):
+        conanfile = """
+from conans import ConanFile
+
+
+class PkgConfigConan(ConanFile):
+    name = "foo"
+    version = "0.1"
+    description = ("Conan, ",
```

Comment:
Is a tuple an allowed description? I don't recall that it is.

Reply:
Not a description, but a python feature. Months ago I watched a python meetup where that usage was more pythonic. And here is the reference: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses.

Reply:
That is without the comma @uilianries 

```python
description = ("Conan, "
                       " the asdasdasd")
```
That will resolve to a single string, not a tuple

Reply:
I see, so it's more a prone error than a bug. Anyway, if we pass a tuple, we have an exception which doesn't help. So we could accept tuple, or raise a better error.

Reply:
Lets make a better error. I don't see here a use case in which we want to support tuples, lists, or other different ways of inputting the data, it should always be a string.

Reply:
Done.